main.go

```diff
@@ -108,7 +108,7 @@ func loadConfig() {
 	config.RegisterDurationConfigVariable(15, &gracefulShutdownTimeout, false, time.Second, "GracefulShutdownTimeout")
 	config.RegisterIntConfigVariable(524288, &MaxHeaderBytes, false, 1, "MaxHeaderBytes")
 
-	config.RegisterStringConfigVariable(enterpriseToken, &enterpriseToken, false, "enterpriseToken")
+	enterpriseToken = config.GetEnv("ENTERPRISE_TOKEN", enterpriseToken)
```

Comment:
I find this a bit confusing since `enterpriseToken` is empty to begin with, maybe do:
```go
	enterpriseToken = config.GetEnv("ENTERPRISE_TOKEN", "")
```

Reply:
enterpriseToken is not always empty, it can be defined as a build arg

Reply:
What I meant is that it's always empty when you are there, on line 111. Right? If not then my bad :slightly_smiling_face:

Reply:
Oh wow :smile: Never seen `ldflags` used to pass credentials before. I guess that there's always a first time.